loops/1_loops.php

```diff
@@ -1,36 +1,32 @@
 <?php
-
 /**
- * for loop
- * @param $countTo
+ * @param {number} countTo sets the boundry imit of loop iteration
  */
 function forLoop($countTo) {
-    for ($i = 1; $i <= $countTo; $i++){
-        print($i . "<br>");
+    for ($i = 0; $i < $countTo; $i += 1){
+        print($i . "\n");
```

Comment:
Można to uprościć jeszcze bardziej.
Jeśli używasz podwójnych cudzysłowów to można do środka wrzucić zmienną bez konieczności konkatenowania z kropkami:

```php
print("$i\n");
```